src/FsAutoComplete/LspServers/AdaptiveServerState.fs

```diff
@@ -307,10 +372,12 @@ type AdaptiveState(lspClient: FSharpLspClient, sourceTextFactory: ISourceTextFac
     let checkUnusedOpens =
       async {
         try
-          use progress = new ServerProgressReport(lspClient)
+          use progress = progressLookup.CreateProgressReport(lspClient, cancellable = true)
```

Comment:
Now creating a `ServerProgressReport` thru `ServerProgressLookup`. Specifying `cancellable = true` allows the button to show.

Reply:
How do these lookups behave when there's no progress token? Can they reasonably be cancelled by a user at that point?

Reply:
`ServerProgressReport` always [creates a `ProgressToken`](https://github.com/ionide/FsAutoComplete/blob/e393d354fb5b167c782a49b66a39b70b48dfe1e2/src/FsAutoComplete/LspServers/FSharpLspClient.fs#L124) otherwise we couldn't send a `$/progress` from the server to the client.